src/Grid/grid.jl

```diff
@@ -709,6 +709,18 @@ end
 @inline getcoordinates(grid::AbstractGrid, cell::CellIndex) = getcoordinates(grid, cell.idx)
 @inline getcoordinates(grid::AbstractGrid, face::FaceIndex) = getcoordinates(grid, face.idx[1])
 
+function cellnodes!(global_nodes::Vector{Int}, grid::AbstractGrid, i::Int)
+    cell = getcells(grid, i)
+    cellnodes!(global_nodes, grid, cell)
+end
+function cellnodes!(global_nodes::Vector{Int}, grid::AbstractGrid, cell::AbstractCell)
+    @assert length(global_nodes) == nnodes(cell)
+    @inbounds for i in 1:length(global_nodes)
+        global_nodes[i] = cell.nodes[i]
+    end
```

Comment:
And to simplify, perhaps:
```suggestion
    copyto!(global_nodes, cell.nodes)
```